HorizonNing/src/main/java/com/hubspot/horizon/ning/NingAsyncHttpClient.java

```diff
@@ -49,7 +54,19 @@ public NingAsyncHttpClient(HttpConfig config) {
 
     this.eventLoopGroup = newEventLoopGroup();
 
-    AsyncHttpClientConfig ningConfig = new DefaultAsyncHttpClientConfig.Builder()
+    DefaultAsyncHttpClientConfig.Builder builder = new DefaultAsyncHttpClientConfig.Builder();
+
+    if (config.isSocksProxied()) {
+      LOG.info(
+        "Client will be routed via SOCKS proxy with host: {} and port: {}",
```

Comment:
nit: I think this message will be clearer without the `:`s after `host` and `port`. You could also do something like `Client will be routed via SOCKS proxy {}:{}`

Reply:
Went ahead and updated. Thanks!